docs/style-guide.rst

```diff
@@ -701,7 +701,65 @@ indistinguishable from the numerals one and zero.
 Contract and Library Names
 ==========================
 
-Contracts and libraries should be named using the CapWords style. Examples: ``SimpleToken``, ``SmartBank``, ``CertificateHashRepository``, ``Player``.
+Contracts and libraries should be named using the CapWords style. Examples: ``SimpleToken``, ``SmartBank``, ``CertificateHashRepository``, ``Player``. Contract and library names should match their Constructor name and should also match their filenames (i.e. if the Contract name is `Congress` and the library name is `Owned`, then their associated Constructor names should be written in CapWords as `Congress` and `Owned` respectively, and their associated filenames should be Congress.sol and Owned.sol)
```

Comment:
> Contract and library names should match their Constructor name

I think this is still wrong. If they don't match, it is just a function and not a constructor :)

Reply:
@axic Yes, I think you're absolutely right, thanks for noticing that. What do others think?

Reply:
So we should change it from `Contract and library names should match their Constructor name and should also match their filenames` to `Contract and Library names must match their Constructor name (otherwise it is just a Function rather than a Constructor), and Contract and Library names should also match their Filenames`

Reply:
I would remove most of that sentence since constructors, by definition, match the contract name and it's unnecessary to repeat that fact here. I would just say:
> Contract and library names should match their filenames.